core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -69,11 +69,11 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec {
   }
 
   "MethodConfigApi" should "return 201 on create method configuration in Agora" in withTestDataApiServices { services =>
-    testCreateMethodConfiguration(AgoraMethod("dsde", "three_step", 1), "three_step", services)
+    testCreateMethodConfiguration(AgoraMethod("dsde", "three_step", 1), "three_step_dockstore", services)
   }
 
   it should "return 201 on create method configuration in Dockstore" in withTestDataApiServices { services =>
-    testCreateMethodConfiguration(DockstoreMethod("dockstore-method-path", "dockstore-method-version"), "three_step_dockstore", services)
+    testCreateMethodConfiguration(AgoraMethod("dsde", "three_step", 1), "three_step", services)
```

Comment:
this should really be testing a `DockstoreMethod`. If for some reason that's not possible (why?),  just delete the test, since it duplicates the one above it

Reply:
Oh shoot, thanks for catching that